frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -17,11 +20,23 @@ export const BrowseSightings: React.FC = () => {
   return (
     <>
       <h1>Reported Sightings</h1>
-      <ul>
-        {sightings.map((sighting, index) => (
-          <SightingCard sighting={sighting} key={sighting.id} />
-        ))}
-      </ul>
+      <button
+        onClick={() => {
+          setMapToggle(!mapToggle);
+        }}
+      >
+        Sightings Map
```

Comment:
Better UX to have the wording change based on which is currently displayed:
```suggestion
        {mapToggle ? "List View" : "Map View"}
```